src/wallet/scriptpubkeyman.h

```diff
@@ -467,4 +468,49 @@ class LegacySigningProvider : public SigningProvider
     bool GetKeyOrigin(const CKeyID& keyid, KeyOriginInfo& info) const override { return m_spk_man.GetKeyOrigin(keyid, info); }
 };
 
+class DescriptorScriptPubKeyMan : public ScriptPubKeyMan
+{
+public:
+    using ScriptPubKeyMan::ScriptPubKeyMan;
+
+    bool GetNewDestination(const OutputType type, CTxDestination& dest, std::string& error) override;
+    isminetype IsMine(const CScript& script) const override;
+
+    bool CheckDecryptionKey(const CKeyingMaterial& master_key, bool accept_no_keys = false) override;
+    bool Encrypt(const CKeyingMaterial& master_key, WalletBatch* batch) override;
+
+    bool GetReservedDestination(const OutputType type, bool internal, CTxDestination& address, int64_t& index, CKeyPool& keypool) override;
+    void ReturnDestination(int64_t index, bool internal, const CTxDestination& addr) override;
+
+    bool TopUp(unsigned int size = 0) override;
+
+    void MarkUnusedAddresses(const CScript& script) override;
+
+    bool IsHDEnabled() const override;
+
+    bool SetupGeneration(bool force = false) override;
+
+    bool HavePrivateKeys() const override;
+
+    void RewriteDB() override;
```

Comment:
ee6980be437c09df6044a5ced7caa91410453027:  `void RewriteDB() override;` is dropped in fd995af379b73b63b78b276b5b8f31d39b2d2b67

Reply:
Dropped